python/nav/web/templates/foundation-5/field.html

```diff
@@ -1,24 +1,24 @@
-{% load crispy_forms_field %}
+{% load forms %}
 
 {% if field.is_hidden %}
   {{ field }}
 {% else %}
   <div id="div_{{ field.auto_id }}"
-       class="ctrlHolder
-       {% if field.errors and form_show_errors %} error{% endif %}
-       {% if field|is_checkbox %} checkbox{% endif %}
-       {% if field.css_classes %} {{ field.css_classes }}{% endif %}">
+       class="ctrlHolder{% if field.errors %} error{% endif %}{% if field.css_classes %} {{ field.css_classes }}{% endif %}"
```

Comment:
Why did you remove "form_show_errors" from the if? Ditto the is_checkbox-check.

Reply:
The css classed that are in use but that do not have a rule should be mentioned in the META-issue, ditto form_show_errors, so that we do the same thing everywhere and document it. I'll start.

Do you think `form_show_errors` *never* has an effect? All over?

Reply:
I grepped, and `form_show_errors` is only ever used directly in templates, never changed in context.